### PR TITLE
Resolve Issue #1. Sidebar fix achieved.

### PR DESCRIPTION
**Pull request by SwittS**
I have corrected <a href="https://github.com/sf-wdi-30/publify_debugging_lab/issues/1">Issue #1</a>. There was a minor typo in the partial content.html.erb for the archives sidebar view. The new fix will make the font monospace and the bullet point unfilled. This fix is successful because there doesn't exist a css class for sidebar_title. There does exist however, a sidebar-title class.

### CODE REVIEW

File: app/controllers/admin/dashboard_controller.rb [hunk 1]
```diff
@@ -17,8 +17,7 @@ def index
     @statsdrafts = Article.drafts.where('created_at > ?', today).count
     @statspages = Page.where('published_at > ?', today).count
     @statuses = Note.where('published_at > ?', today).count
-    @statuserposts = Article.published.where('published_at > ?', today).count(conditions: { user_id: current_user.id })
-    @statcomments = Comment.where('created_at > ?', today).count
+    @statuserposts = current_user.articles.published.where('published_at > ?', today).count @statcomments = Comment.where('created_at > ?', today).count
```
Comment (justincastilla) [inline app/controllers/admin/dashboard_controller.rb]: How is this change relevant to the issue?

**Comment by SwittS** [comment]
This fix is successful because there doesn't exist a css class for sidebar_title. There does exist however, a sidebar-title class. 
You're right, the dashboard_controller code is not relevant to the current issue and does not belong in this pull request.

**Comment by justincastilla** [comment]
Please remove the unnecessary code change. Also include your code explanation in your original pull request message.

**Comment by SwittS** [comment]
Removed unnecessary code change and included code explanation in the original pull request message!
